app/models/github.rb

```diff
@@ -16,6 +16,28 @@ def self.update_lessons(params)
     update_removed_lessons(params[:repo], params[:removed]) if params[:removed].try(:any?)
   end
 
+  def self.build_section(section)
```

Comment:
Why not put this in the Section model?

Reply:
I moved it back and forth a couple times but felt like the code is so specific to Github that it made more sense to put in the Github class. I'm happy to move it though if you think it would be better there. I also was wondering if maybe I can split that method up to put the more Github-code specific part in the Github class and other part in the Section class.

Reply:
Yeah, I like that idea. Use this class to pull the file from Github, use the other class to parse the file and create the section and whatnot.